admin/tests/integration/components/users/certification-center-memberships_test.js

```diff
@@ -1,16 +1,21 @@
 import { module, test } from 'qunit';
-import { setupRenderingTest } from 'ember-qunit';
 import { render, clickByName } from '@1024pix/ember-testing-library';
 import { hbs } from 'ember-cli-htmlbars';
-import EmberObject from '@ember/object';
 import Service from '@ember/service';
 import sinon from 'sinon';
+import setupIntlRenderingTest from '../../../helpers/setup-intl-rendering';
 
 module('Integration | Component | users | certification-center-memberships', function (hooks) {
-  setupRenderingTest(hooks);
+  setupIntlRenderingTest(hooks);
+
+  let store;
+
+  hooks.beforeEach(function () {
+    store = this.owner.lookup('service:store');
+  });
 
   module('When user isn’t member of any certification center', function () {
-    test('it should display an empty table', async function (assert) {
+    test('it displays an empty table', async function (assert) {
```

Comment:
```suggestion
    test('displays an empty table', async function (assert) {
```
Cette suggestion est à appliquer sur les autres modifications.